src/System.ComponentModel.TypeConverter/src/System/ComponentModel/EventHandlerList.cs

```diff
@@ -27,40 +22,33 @@ public EventHandlerList()
         }
 
         /// <devdoc>
-        ///     Creates a new event handler list.  The parent component is used to check the component's
-        ///     CanRaiseEvents property.
-        /// </devdoc>
-        [SuppressMessage("Microsoft.Security", "CA2122:DoNotIndirectlyExposeMethodsWithLinkDemands")]
-        internal EventHandlerList(Component parent)
-        {
-            this.parent = parent;
-        }
-
-        /// <devdoc>
         ///    <para>Gets or sets the delegate for the specified key.</para>
         /// </devdoc>
-        public Delegate this[object key] {
+        public Delegate this[object key]
+        {
             [SuppressMessage("Microsoft.Security", "CA2122:DoNotIndirectlyExposeMethodsWithLinkDemands")]
-            get {
+            get
+            {
                 ListEntry e = null;
-                if (parent == null || parent.CanRaiseEventsInternal)
```

Comment:
We removed `parent` entirely because it's a `Component` (unavailable in CoreFX).